src/components/QuestionnaireMessage.js

```diff
@@ -0,0 +1,44 @@
+import Modal from "@mui/material/Modal";
+import * as s from "./ExtensionMessage.sc";
+import Feature from "../features/Feature";
+import LocalStorage from "../assorted/LocalStorage";
+import strings from "../i18n/definitions";
+
+export default function QuestionnaireMessage({
+  open,
+  setDisplayedAudioExperimentQuestionnaire,
+  setQuestionnaireMessageOpen,
+}) {
+  function handleClose() {
+    setQuestionnaireMessageOpen(false);
+    setDisplayedAudioExperimentQuestionnaire(true);
+    LocalStorage.setDisplayedAudioExperimentQuestionnaire(true);
```

Comment:
maybe log the event also to the server? in case they close the popup without doing the survey you can remind them with an email later.

Reply:
Implented log, so we can see that they closed it.

Reply:
Implented log, so we can see that they closed it.